impl/maven-impl/src/main/java/org/apache/maven/impl/model/DefaultInterpolator.java

```diff
@@ -263,10 +263,9 @@ private static String doSubstVars(
         // Using the start and stop delimiter indices, extract
         // the first, deepest nested variable placeholder.
         String variable = val.substring(startDelim + DELIM_START.length(), stopDelim);
-        String org = variable;
 
         String substValue = processSubstitution(
-                variable, org, cycleMap, configProps, callback, postprocessor, defaultsToEmptyString);
```

Comment:
yes, should be covered by https://pmd.github.io/pmd/pmd_rules_java_bestpractices.html#unusedassignment

Reply:
seems its currently renaming not removal.